authenticator/authenticator.go

```diff
@@ -124,7 +125,7 @@ func (a *Authenticator) GetPGMD5Hash(ctx context.Context, req *pb.PGMD5HashReque
 		return nil, status.Errorf(codes.InvalidArgument, msg)
 	}
 
-	return &pb.PGMD5Response{Hash: computePGMD5(dbuser, password, salt)}, nil
+	return &pb.PGMD5Response{Hash: computePGMD5Hash(dbuser, password, salt)}, nil
```

Comment:
this is just changing the function name to be more consistent with the other SCRAM function names.